src/site/_includes/components/Instruction.js

```diff
@@ -138,13 +137,13 @@ module.exports = (type, listStyle = 'ul') => {
     case 'devtools-application':
     case 'devtools-security':
     case 'devtools-lighthouse':
-      instruction = html`${shared.devtools}`;
-      substitution = type.substring('devtools-'.length);
-      if (substitution) {
-        // prettier-ignore
+      instruction = getInstruction('devtools.open');
+      let tab = type.substring('devtools-'.length);
+      if (tab) {
+        tab = capitalize(tab);
         instruction = html`
-          ${instruction}
-          ${bullet}Click the **${capitalize(substitution)}** tab.
+          ${getInstruction('devtools.open')}
+          ${getInstruction('devtools.open_tab').replace(/<TAB>/g, tab)}
```

Comment:
this seems a bit confusing because `instruction` gets redefined (whereas in the old version it was reused).

Should we remove line 140?